### ISSUE TITLE
`is_stop` depends on capitalisation

### ISSUE THREAD

**Issue by simonm3** [explosion/spaCy#3368]
#1889 was marked as fixed. however I still see the same error:

```
print(spacy.about.__version__)
nlp = spacy.load('en')
for s in ["The store", "the store"]:
    doc = nlp(s)
    print('\n{}'.format(s))
    for t in doc:
        print('{}\t{}'.format(t.text, t.is_stop))
```
2.0.12

The store
The	False
store	False

the store
the	True
store	False

**Comment by ines** [comment]
The `is_stop` property is cached in the models, so this change will only be effective when we train new models.

The upcoming version and models (already available via `spacy-nightly`) were trained after the fix was implemented and should now include case-insensitive stop words. Alternatively, if you use a blank model / language class, you should also see the changes reflected:

```python
from spacy.lang.en import English
nlp = English()
```

**Comment by simonm3** [comment]
Not tried the nightly but the other suggestions don't seem to work for me. Exactly the same results.

**Comment by ines** [comment]
Ohh I think I remember why: Pretty sure we decided to not push this fix into the v2.0.x branch because it's kind of a breaking change that leads to significantly different output of a built-in property. That's not really suitable for a patch release.

Just tried it in v2.1.0 (`spacy-nightly`) and can confirm it works as expected:

```
The store
The     False
store   False

the store
the     True
store   False
```
